# --- ROCK-COPYRIGHT-NOTE-BEGIN ---
# 
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
# Please add additional copyright information _after_ the line containing
# the ROCK-COPYRIGHT-NOTE-END tag. Otherwise it might get removed by
# the ./scripts/Create-CopyPatch script. Do not edit this copyright text!
# 
# ROCK Linux: rock-src/package/rene/cdrdao/linux-asm-byteorder.patch
# ROCK Linux is Copyright (C) 1998 - 2005 Clifford Wolf
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# 
# --- ROCK-COPYRIGHT-NOTE-END ---

diff -dur cdrdao-1.1.9/scsilib/libscg/scsi-linux-sg.c cdrdao-1.1.9-p/scsilib/libscg/scsi-linux-sg.c
--- cdrdao-1.1.9/scsilib/libscg/scsi-linux-sg.c	2004-06-07 01:04:12.000000000 +0000
+++ cdrdao-1.1.9-p/scsilib/libscg/scsi-linux-sg.c	2004-10-25 17:53:24.944188560 +0000
@@ -69,6 +69,19 @@
 #if LINUX_VERSION_CODE >= KERNEL_VERSION(2, 6, 0)
 	#define __KERNEL__
 	#include <asm/types.h>
+
+	#ifdef __CHECKER__
+	#define __bitwise __attribute__((bitwise))
+	#else
+	#define __bitwise
+	#endif
+	#define __le16 __bitwise __u16
+	#define __be16 __bitwise __u16
+	#define __le32 __bitwise __u32
+	#define __be32 __bitwise __u32
+	#define __le64 __bitwise __u64
+	#define __be64 __bitwise __u64
+	
 	#include <asm/byteorder.h>
 	#undef __KERNEL__
 #endif
Nur in cdrdao-1.1.9-p/scsilib/libscg: scsi-linux-sg.c~.
